Begin profile create

RCFonken opened the pull request:
designed profile create view, get en post route aangemaakt, een profile controller aangemaakt, app.blade aangepast voor de main CSS van de applicatie en de favicon van de site aangepast

RCFonken committed:
Begin profile create

designed profile create view, get en post route aangemaakt, een profile controller aangemaakt, app.blade aangepast voor de main CSS van de applicatie en de favicon van de site aangepast

RCFonken committed:
Merge branch 'main' into ProfileCreate

RCFonken committed:
DB functionality

Start met het koppelen van de Db aan het frontend, samen met een start van het samen voegen van een create en update

RCFonken committed:
final changes profile create/update page

- de pagina word nu gebruikt voor create en update van een user profile met een 1 op 1 relatie
- de profielfoto heeft bij update nu een button en een preview van de oude gekozen foto
- alles update nu ook op de juiste manier in de database
-url update van /profile/create naar alleen /profile voor het gemak

De pagina is nu compleet met error berichten/succes berichten, juiste DB connecties en informatie + de juiste authentificatie.

Pluers committed:
Merge branch 'main' into ProfileCreate

Pluers commented:
Ik krijg errors dat de profile.store niet bestaat, en ik weet niet echt naar welke pagina moet

RCFonken committed:
route fix

RCFonken committed:
Merge branch 'ProfileCreate' of https://github.com/Djimairo7/F1Registration into ProfileCreate

Pluers committed:
Merge branch 'main' into ProfileCreate

Pluers committed:
Update profile creation and editing

This commit updates the profile creation and editing functionality. It includes changes to the migration file for the profiles table, the dashboard view, the web routes, and the ProfileController. The changes allow users to create and edit their profiles, including uploading a profile picture.
